python/lsst/pipe/tasks/postprocess.py

```diff
@@ -78,6 +83,96 @@ def flattenFilters(df, noDupCols=["coord_ra", "coord_dec"], camelCase=False, inp
     return newDf
 
 
+class TableVStack:
```

Comment:
This is great, and I want to make sure it's used as widely as possible.  Is this the best place for this code?

Reply:
I think it's fine - all current usage is in this file, and I think other usage is more likely to be in `pipe_tasks` than upstream of it.